enable/tools/pyface/api.py

```diff
@@ -6,20 +6,6 @@
 # LICENSE.txt
 #
```

Comment:
You're renaming an `api` module here in a pretty significant way. Perhaps it would be better to leave the old module behind and import from the new module [Maybe even raise a warning for code which is importing from the old module]? @corranwebster How kind are we being to downstream code with changes like this? Do you have a preference in any direction?

Reply:
We're being kind, but this is a major release.  I'd suggest keeping `apptools/api.py` around, but import from the pyface `api.py`.

Reply:
Oh, and raise appropriate deprecation warnings for importing from `enable.tools.apptools.api`